docs/releases/v0.1.1.md

```
@@ -0,0 +1,24 @@
+# Release v0.1.1
+
+Release date: 2026-02-28
+Type: stable (`latest`)
+Package: `codex-multi-auth`
+
+### Highlights
```

markdownlint-cli2 (0.21.0)

[warning] Heading levels should only increment by one level at a time
Expected: h2; Actual: h3

(MD001, heading-increment)